web-components/button/button.svelte

```diff
@@ -21,19 +24,24 @@
 
 <style lang="scss">
   // Main styles and states
-  .leoButton {
+  .leoButton,
+  a.leoButton {
     --box-shadow-hover: var(--effect-elevation-02);
+    display: block;
     cursor: pointer;
+    position: relative;
     transition: box-shadow .12s ease-in-out, color .12s ease-in-out;
     box-shadow: none;
     border: solid var(--border-width, 0px) var(--border-color, transparent);
     border-radius: var(--radius-full);
     background: var(--bg);
     color: var(--color);
-    &:enabled {
+    &:not(:disabled) {
```

Comment:
For my own curiosity, how is `:not(:disabled)` different to `:enabled`. Sorry for the dumb questions :laughing:

Reply:
Not dumb at all! I was confused on this point too. Anchor tags can't be enabled/disabled, so selecting the `&:enabled` selector breaks when the button uses `<a>`.

Reply:
Right, that makes sense!